acceptance/variables/complex/script

```diff
@@ -0,0 +1,4 @@
+$CLI bundle validate -o json | jq '{resources,variables}' > out.default.json
```

Comment:
since it's a bash script, shall we better name it `script.sh` to make it more clear?

Reply:
I don't know if 'bash' is going to stay or the only thing that's used. 

I think we might want to support other runners, especially for interactive use case. For example, expect or pexpect https://pexpect.readthedocs.io/en/stable/overview.html

Reply:
Idea: We could use the extension to determine the runner? `script.sh` vs `script.py`?

Reply:
We could still do that once we have add another runner. script for current bash case and script.py for python runner.